patches/third_party-blink-public-common-BUILD.gn.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/third_party/blink/public/common/BUILD.gn b/third_party/blink/public/common/BUILD.gn
+index 64c1801272f78add24d74d5d225de9cd9085b420..6a270770fa981109fdc52f20d7ffaf41d9c1680e 100644
+--- a/third_party/blink/public/common/BUILD.gn
++++ b/third_party/blink/public/common/BUILD.gn
+@@ -33,6 +33,7 @@ buildflag_header("buildflags") {
+ 
+ source_set("headers") {
+   visibility = [
++    "//brave/components/speedreader:speedreader",
```

Comment:
If we can't open this to  "//brave/components/speedreader:speedreader", please let me know.